plugins/SBOMVis/pyproject.toml

```
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "surfactant-SBOMVis"
```

```suggestion
name = "sbomvis"
```

Let's make the name sbomvis -- once Surfactant gets the ability to read CycloneDX/SPDX SBOM files as input then the visualizer will be pretty close to usable with SBOMs generated by other tools.

Description can be updated later.